frontend/util.lua

```diff
@@ -94,4 +96,29 @@ function util.lastIndexOf(string, ch)
     if i == nil then return -1 else return i - 1 end
 end
 
+
+-- Split string into a list of UTF-8 chars.
+-- @text: the string to be splitted.
+-- @tab: the table to store the chars sequentially, must not be nil.
+function util.splitToChars(text, tab)
```

Comment:
Any reason why you prefer clearing a passed in table to creating and returning a new table inside the function?